devtools/x/src/config.rs

```diff
@@ -26,6 +27,8 @@ pub struct Config {
     /// Cargo configuration
     cargo: CargoConfig,
     grcov: CargoTool,
+    /// Determinator configuration
+    determinator: DeterminatorRules,
```

Comment:
Hope these have sane defaults that capture the standard cargo setup :D

Reply:
You know, I've been meaning to add defaults for e.g. `rust-toolchain` etc. Should do them upstream.

Reply:
Done, added defaults upstream.